test/unit/annotation_spec.js

```diff
@@ -4082,7 +4082,7 @@ describe("annotation", function () {
         const popupDict = new Dict();
         popupDict.set("Type", Name.get("Annot"));
         popupDict.set("Subtype", Name.get("Popup"));
-        popupDict.set("F", 25); // not viewable
+        popupDict.set("F", 56); // not viewable
```

Comment:
Does the unit-test still "make sense" with these changes, i.e. does this updated flag still equal `not viewable` as the comment says?

Reply:
I removed 1 (Invisible) and added 32 (NoView) so it should be `not viewable`.